atlasdb-autobatch/src/main/java/com/palantir/atlasdb/autobatch/DisruptorAutobatcher.java

```diff
@@ -36,10 +38,19 @@
  */
 public final class DisruptorAutobatcher<T, R>
         implements AsyncFunction<T, R>, Function<T, ListenableFuture<R>>, Closeable {
-    private static final int BUFFER_SIZE = 1024;
+
+    /*
+        By memoizing thread factories per loggable purpose, the names are correct for multiple instances of the same
+        autobatcher function.
```

Comment:
I agree. Would be worth saying what `correct` means, though (i.e. threads are numbered uniquely across different instances)